opentelemetry-sdk/src/opentelemetry/sdk/trace/__init__.py

```diff
@@ -58,13 +58,16 @@
 
 logger = logging.getLogger(__name__)
 
+_DEFAULT_SPAN_EVENTS_LIMIT = 128
+_DEFAULT_SPAN_LINKS_LIMIT = 128
+_DEFAULT_SPAN_ATTRIBUTES_LIMIT = 128
+
 SPAN_ATTRIBUTE_COUNT_LIMIT = int(
```

Comment:
This is not being used anywhere anymore but removing it would break the Python interface of this module. This is strictly a read only variable as over-writing it has no effect at all in the current implementation. I'm not sure what to do with this without completely re-writing this whole module as a class or override __class__ on module which doesn't work with 3.6. 

https://stackoverflow.com/questions/922550/how-to-mark-a-global-as-deprecated-in-python